frontend/src/Pages/Customer/BuyerCatalog.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import axios from 'axios';
+import { Row, Col, Button, Container, Jumbotron, Card, Image, ListGroup } from 'react-bootstrap';
+import ROUTES from '../../routes';
+
+const CALL_BUTTON_IMAGE_URL = "https://secure.webtoolhub.com/static/resources/icons/set112/56bb7adb.png";
```

Comment:
We shouldn't load images from random websites, can we replace this with something that is available natively ( store the png in our code).

Also make sure whatever images/icons we are using, we have the rights to use the image.

Reply:
Fixed this. Changed the Icon to an MIT License one and stored it natively